src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projecteditor/view/ProjectEditor.kt

```diff
@@ -64,23 +65,25 @@ class ProjectEditor : View() {
         addClass(AppStyles.appBackground)
         hbox {
             vbox {
-                label {
-                    textProperty().bind(viewModel.projectTitleProperty)
-                    addClass(ProjectEditorStyles.projectTitle)
-                }
-                childrenList = listview {
-                    items = viewModel.children
+                vgrow = Priority.ALWAYS
+                childrenList = projectnav(
+                        viewModel.projectProperty,
+                        viewModel.activeChildProperty,
+                        viewModel.activeContentProperty) {
```

Comment:
passing the view model properties doesn't feel like the right way to solve this problem (looking to other UI controls in tfx, we bind properties there in a different way), though this problem may have come from trying to adhere to the constraint of running as an extension method, so the refactor may just naturally fix this